build/ReleaseManager/osg-release.sh

```diff
@@ -33,7 +33,7 @@ fi
 # On August 2024 OSG switched to osg-sw-submit.chtc.wisc.edu. moria and library.cs.wisc.edu are not working any more
 #osg_buildmachine="osg-sw-submit.chtc.wisc.edu"
 # On March 22 2025 the build host has been moved to osg-sw-submit-old
-osg_buildmachine="osg-sw-submit-old.chtc.wisc.edu"
+osg_buildmachine="osg-sw-submit.chtc.wisc.edu"
```

Comment:
Has the OSG build machine been reverted back to what the name was before the `-old` suffix was used?

Reply:
yes